.github/workflows/ci.yaml

```diff
@@ -54,8 +54,14 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - name: Install Helm Docs
+        uses: envoy/install-helm-docs@v1.0.0
+        with:
+          version: 1.11.0
       - name: Run helm-docs
-        run: .github/helm-docs.sh
```

Comment:
This script previously handled both installation and running of `helm-docs`, more than one contributor (and myself when I forget...) have run the script locally only to have the project get piles of `helm-docs` binaries installed.

This seemed cleaner, and some tooling can be added (probably a Makefile or little script) to help with local changes.